backend/tracim_backend/models/event.py

```diff
@@ -19,6 +20,32 @@
 from tracim_backend.models.meta import DeclarativeBase
 
 
+class EventType(object):
```

Comment:
The naming is confusing: this is not an event type but a utility class.
I would also rather have a static method to convert from event_type -> database parameters:
```python
class EventTypeDatabaseParameters:
    def __init__(self, entity: EntityType, operation: OperationType, subtype: typing.Optional[str]) -> None:
        self.entity = entity
        self.operation = operation
        self.subtype = subtype
   
    @staticmethod
    def from_event_type(event_type: str) -> "EventTypeDatabaseParameters":
        ...
        return EventTypeDatabaseParameters(entity, operation, subtype)
```
This makes easier for a reader to see which attributes are set on an instance as the only code in `__init__` is setting them.

Reply:
done.